<launch>
    <group ns="YawRatePidCtrlParams">
        <param name="kp" type="double" value="3.0"/>
        <param name="kd" type="double" value="0.0"/>
        <param name="ki" type="double" value="0.0"/>
    </group>
    <group ns="PitchRatePidCtrlParams">
        <param name="kp" type="double" value="3.0"/>
        <param name="kd" type="double" value="0.0"/>
        <param name="ki" type="double" value="0.0"/>
    </group>
    <group ns="RollRatePidCtrlParams">        
        <param name="kp" type="double" value="3.0"/>
        <param name="kd" type="double" value="0.0"/>
        <param name="ki" type="double" value="0.0"/>
    </group>
    <node pkg="gimbal_control" type="gimbal_control" name="gimbal_control" output="screen" cwd="node">
        <param name="gimbal_control_time_step_sec" type="double" value="0.02" />
    </node>


</launch>



